<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QT_classifier_trainClass</class>
 <widget class="QMainWindow" name="QT_classifier_trainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;blackboard.jpg&quot;)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>770</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;&quot;);
color:#FFFFFF</string>
    </property>
    <property name="text">
     <string>訓練影像資料夾路徑</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>530</y>
      <width>370</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;&quot;);
color:#FFFFFF</string>
    </property>
    <property name="text">
     <string>訓練次數</string>
    </property>
   </widget>
   <widget class="QPushButton" name="path_button">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>460</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image:url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>選擇路徑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>800</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image:url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>開始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>740</width>
      <height>170</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;&quot;);
color:#FFFFFF</string>
    </property>
    <property name="text">
     <string>模型檔名稱(限英文)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>1600</x>
      <y>925</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微軟正黑體</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;
background-image:url(&quot;&quot;)</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pb_editline">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>1200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_editline">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>1200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="train_editline">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>690</y>
      <width>1200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border: 3px solid #FFFFFF;
color: #FFFFFF;</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>path_button</zorder>
   <zorder>start</zorder>
   <zorder>return_button</zorder>
   <zorder>pb_editline</zorder>
   <zorder>path_editline</zorder>
   <zorder>train_editline</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
